Resources/Prototypes/DeltaV/Entities/Markers/Spawners/jobs.yml

```diff
@@ -24,8 +24,8 @@
     layers:
       - state: green
       - state: lawyer
```

Comment:
` - state: lawyer` needs to be removed from each of these. It was just a placeholder

Reply:
This is an outdated file, it was already resolved